content/source/assets/stylesheets/_docs.scss

```diff
@@ -61,18 +108,21 @@
     padding-top: 10px;
 
     li {
+      padding: 0 0 11px 0;
+
       a {
         @extend %sidebar-link;
+        padding-left: 6px;
+        display: table-cell;
       }
 
       &:before {
         color: $sidebar-link-color-active;
         content: '\2022';
         font-size: 1em;
-        left: -3px;
-        top: -3px;
         opacity: 0.4;
-        position: absolute;
+        position: relative;
+        display: table-cell;
```

Comment:
This took me a bit of inspector fiddling to understand. I think we're using `table-cell` here (and on the `a`) to maintain a hanging indent between the custom bullet and the text, for long sidebar page titles. Is that correct?

If so, maybe a comment would be helpful.

Reply:
That's exactly right, and I definitely should have left a comment there. Thanks for fiddling, I did this one commit in April and it probably would have taken me just as much fiddling to remember why I did that.